### PR TITLE
Web header

### PR DESCRIPTION
**Pull request by mumbi58**
# Description

I've successfully implemented a website header, addressing the need for clear navigation and improved user experience. This change was driven by the goal of enhancing site aesthetics and functionality. The header serves as a pivotal element for user orientation and engagement, necessitating the availability of design assets, including the logo and navigation links, while also ensuring compatibility with the chosen web development framework or CMS.
#1 
## Type of change
- [ ] New feature (non-breaking change which adds functionality)
## Screenshot 
laptop view
image
phone view 
image

##TEST
I used the browser's "Inspect" option to assess the responsiveness of the website header on various devices, ensuring that it displayed correctly and functioned as intended
## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (paulkahura) [inline src/css/style.css]: remove extra spaces

**Review by paulkahura — CHANGES_REQUESTED** [review]
Nice job! Just make sure to use rem/em over px.Also check the contact us button padding

File: src/css/style.css (hunk 1 above)
Comment (mumbi58) [reply src/css/style.css]: Thank you for the feedback. i have made the changes required

**Review by JAPHETHNYARANGA — COMMENTED** [review]
closing as this is done

**Pull request closed without merging**